q2_qemistree/_itol_metadata.py

```diff
@@ -0,0 +1,67 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2018, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+import pandas as pd
+import seaborn as sns
+import click
+from qiime2 import Artifact
+
+
+def classyfire_to_colors(classified_feature_data: pd.DataFrame,
+                         classyfire_level: str, color_palette: str):
+    '''This function generates a color map (dict) for unique Classyfire
+    annotations in a user-specified Classyfire level.'''
+    color_map = {}
+    annotations = classified_feature_data[classyfire_level].unique()
+    colors = sns.color_palette(color_palette,
```

Comment:
Do you think we should raise a warning if the number of unique annotations is larger than the number of available colors? Or is something like this evident from iTOL?